### PR TITLE
feat: add backup tool script

### PR DESCRIPTION
**Pull request by Ja7ad**
## Description

This script make backup from node database.

### CODE REVIEW

File: scripts/backup/backup_tool.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+        }
+
+        return {"data": file_info}
+
+
+class DaemonManager:
+    @staticmethod
+    def is_daemon_running(daemon_path):
+        process_name = os.path.basename(daemon_path)
+        if os.name == 'nt':
```
Comment (b00f) [inline scripts/backup/backup_tool.py:107]: I don't see any point supporting windows here

File: scripts/backup/backup_tool.py [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                if os.name == 'nt':
+                    subprocess.run(['taskkill', '/F', '/IM', process_name], timeout=10)
+                else:
+                    pids = subprocess.run(['pgrep', '-f', daemon_path], capture_output=True,
+                                          text=True).stdout.strip().split()
+                    current_pid = os.getpid()
+                    pids_to_kill = [pid for pid in pids if pid != str(current_pid)]
+                    logging.debug(f"PIDs to kill: {pids_to_kill}")
+                    for pid in pids_to_kill:
+                        subprocess.run(['kill', pid], timeout=10)
```
Comment (b00f) [inline scripts/backup/backup_tool.py:133]: Can we send interrupt signal here?

File: scripts/backup/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Pactus Blockchain Backup tool
```
Comment (b00f) [inline scripts/backup/README.md:1]: Please add this comment to the header of the script.

File: scripts/backup/backup_tool.py [hunk 4]
```diff
@@ -0,0 +1,296 @@
+import argparse
```
Comment (b00f) [inline scripts/backup/backup_tool.py:1]: Let's rename to `snapshot.py` and move it to `./scripts`

**Review by b00f — CHANGES_REQUESTED** [review]

File: scripts/backup/backup_tool.py (hunk 1 above)
Comment (Ja7ad) [reply scripts/backup/backup_tool.py:107]: `nt` is windows otherwise it is unix base and linux

File: scripts/backup/backup_tool.py (hunk 4 above)
Comment (Ja7ad) [reply scripts/backup/backup_tool.py:1]: `scripts` folder is for any script.

I think it's better not to move it.

**Review by b00f — CHANGES_REQUESTED** [review]

File: scripts/snapshot.py [hunk 5]
```diff
@@ -0,0 +1,338 @@
+# MIT License
+#
+# Copyright (c) 2024 Javad Rajabzadeh
```
[the remark this answers is not in the document]
Comment (Ja7ad) [reply scripts/snapshot.py:3]: This is a third party tool and I wrote it.
The license is personal for this script.

**Review by b00f — APPROVED** [review]

**Review by b00f — APPROVED** [review]

**Pull request merged**